src/frontend/apps/bbb/DashboardMeetingLayout/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { Box, Grid, ResponsiveContext, Text } from 'grommet';
+import React, { useContext } from 'react';
+
+import { appData } from 'data/appData';
+import { Nullable } from 'utils/types';
+
+interface DashboardMeetingMessageProps {
+  message: string;
+}
+
+export const DashboardMeetingMessage = ({
+  message,
+}: DashboardMeetingMessageProps) => (
+  <Box
+    margin={{ top: 'xlarge', horizontal: 'small' }}
+    pad={{ vertical: 'small', horizontal: 'small' }}
+    background="blue-message"
+    round="xsmall"
+  >
+    <Text color="blue-active" textAlign="center" weight="bold">
+      {message}
+    </Text>
+  </Box>
+);
+
+interface DashboardMeetingLayoutProps {
+  left: JSX.Element;
+  right?: Nullable<JSX.Element>;
+}
+
+export const DashboardMeetingLayout = ({
+  left,
+  right,
+}: DashboardMeetingLayoutProps) => {
```

Comment:
I see no test for this component. You should test with controlled left and right component